src/node_dotenv.cc

```diff
@@ -8,6 +9,12 @@ namespace node {
 using v8::NewStringType;
 using v8::String;
 
+std::regex LINE(
```

Comment:
Can you add a description with the link to the original implementation?

Reply:
Done